keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -17,9 +17,27 @@
 
 import tensorflow as tf
 import tensorflow_text as tf_text
+from tensorflow import keras
 
 from keras_nlp.tokenizers import tokenizer
 
+# Pretrained Vocabularies
+
+BASE_PATH = "https://storage.googleapis.com/keras-nlp/pretrained-tokenizers/wordpiece-tokenizer/vocabularies/"
```

Comment:
We should store hashes for all the file downloads we provide. Take a look at how keras applications does this